userprofile/templates/userprofile/profile.html

```diff
@@ -12,62 +12,60 @@
 {% endblock %}
 
 {% block content %}
-<div class="page-header">
-    <h1>
-        {% if user.first_name or user.last_name %}
-        {{ user.first_name }} {{ user.last_name }}
-        {% else %}
-        {% trans "[Unknown name]" %}
-        {% endif %}
-    </h1>
+<div class="row">
+	<div class="col-md-offset-2 col-md-2 page-header">
+		<div class="gravatar">
+			<a href="http://gravatar.com" target="_blank" rel="noopener noreferrer">
```

Comment:
I can remove it. However, I would like to argue that I do not know a single website that does not provide an avatar or profile picture. I am sure that there is not much value added by this feature, but it seems to be a standard. Do we really want to remove the icon/picture?

Regarding the wide margins, I added the `col-md-offset-2` property. If you visit websites such as GitHub, Facebook,  StackOverflow, among many others, you can see that when the content is shown in really wide screens the page make use of wide margins and centred content. If I am correct, that will improve the readability for certain users. Of course, if they use the zoom the text will adapt and the margins will be removed.

https://www.w3.org/TR/low-vision-needs/#margins-and-borders
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html

Reply:
The user profile avatars are used so little in A+ that people hardly notice that they exist at all. However, they depend on external Wordpress.com accounts, which is not nice from the modern privacy perspective. Removing the icons completely is the easiest solution. It is not worthwhile right now to develop a new profile icon feature in A+.

Wide margins: I know many web sites use them, but A+ hasn't. If the profile page is the only page in A+ that uses wide margins, then it is not consistent with the rest of the system. Since I am not used to seeing wide margins in A+, my initial reaction was that it looked unusual and inconsistent. We can talk more about this later.